package.json

```diff
@@ -15,7 +17,12 @@
   },
   "homepage": "https://github.com/RoyNyaga/microverse_restaurant_page_js#readme",
   "devDependencies": {
-    "eslint": "^6.7.2",
+    "eslint": "^6.1.0",
+    "eslint-config-airbnb": "^18.0.1",
+    "eslint-plugin-import": "^2.19.1",
+    "eslint-plugin-jsx-a11y": "^6.2.3",
+    "eslint-plugin-react": "^7.17.0",
+    "eslint-plugin-react-hooks": "^1.7.0",
```

Comment:
These are `React` specific Eslint plugin. I recommend you remove them as they are useless for this project.